pyproject.toml

```diff
@@ -27,4 +27,24 @@ exclude = '''
   | docs/conf.py
   | _astropy_init.py
 )
-'''
\ No newline at end of file
+'''
+
+[tool.interrogate]
+color = true
+#exclude = ["setup.py", "docs", "build"]
+#fail-under = 90
+ignore-init-method = true
+ignore-init-module = true
+ignore-magic = true
+ignore-module = true
+ignore-nested-functions = false
+ignore-nested-classes = true
+ignore-private = true
+ignore-property-decorators = true
+#ignore-regex = ["^get$", "^mock_.*", ".*BaseClass.*"]
+ignore-semiprivate = true
+ignore-setters = true
+omit-covered-files = false
+quiet = false
+verbose = 0
```

Comment:
I assume this matches our existing coverage settings?

Reply:
Good question. Our previous configuration was:

```yaml
paths:
  - tardis
#badge: docs
#exclude: .*/test # regex
verbose: 2 # int (0-3)
skip_magic: True
skip_file_doc: True
skip_init: True
skip_class_def: False
skip_private: True
follow_links: True
#ignore_names_file: .*/test # regex
#fail_under: 90
percentage_only: False
#ignore_patterns: # Dict with key/value pairs of file-pattern/node-pattern
#  .*: method_to_ignore_in_all_files
#  FileWhereWeWantToIgnoreAllSpecialMethods: "__.+__"
#  SomeFile:
#    - method_to_ignore1
#    - method_to_ignore2
#    - method_to_ignore3
#  a_very_important_view_file:
#    - "^get$"
#    - "^set$"
#    - "^post$"
#  detect_.*:
#    - "get_val.*"
```

Seems quite similar.

The actual coverage with `interrogate` is `50.08` and the previous one is `49.65`. What's your opinion on this?

Reply:
Close enough for me!